lib/hyrax/configuration.rb

```diff
@@ -83,41 +158,239 @@ def registered_user_group_name
     end
 
     # @!endgroup
+    # @!group Identifier Minting
+
+    attr_writer :enable_noids
+    def enable_noids?
+      return @enable_noids unless @enable_noids.nil?
+      @enable_noids = true
```

Comment:
This seems to be an unexpected change in state.

Reply:
this code is just moved from below. not changed.